contextutil/context.go

```diff
@@ -47,13 +49,19 @@ func Tenant(ctx context.Context) (config.TenantConfiguration, error) {
 }
 
 // CentContext updates a context with tenant info using the configstore, must only be used for api handlers
-func CentContext(ctx context.Context, config config.Service) (context.Context, error) {
-	// TODO [multi-tenancy] remove following and read the tenantID from the context
-	tc, err := config.GetAllTenants()
+func CentContext(ctx context.Context, cs config.Service) (context.Context, error) {
```

Comment:
NodeContext instead of CentContext?

Reply:
Sure, will rename soon in the other pr I have for config stuff. I want to have this in dev and use it in my other prs.